.browserslistrc

```diff
@@ -0,0 +1,3 @@
+> 1%
```

Comment:
Is this enough coverage?

Reply:
@AbigailMcP @edwardhorsford Are is the `browserslistrc` ok?

Reply:
How are we deciding on these vs the `browserlist` default options of `> 0.5%, last 2 versions, Firefox ESR, not dead`? Specifically, why do we support `> 1%` rather than `> 0.5%`?

Also, do we need to specifically add `ie 11`? When I run `npx browserslist` with and without it, it's still in the list of supported browsers. Is it to make doubly sure it's included?

Reply:
@jhenderson the current GDS set of [recommended browsers is here](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices).

Reply:
@AbigailMcP I got the browserlist from a different website. I'm going to copy the default list but add on `ie 11`.

@edwardhorsford The new updated browserlist should cover the GDS set.

Reply:
👍